src/tools/devtools/release-notes/index.md

```diff
@@ -1,11 +1,17 @@
 ---
 title: DevTools release notes
-short-title: DevTools release notes
-description: Release notes for Flutter and Dart DevTools
+description: Learn about the latest changes in Dart and Flutter DevTools.
+toc: false
 ---
 
-This page links to announcements and release notes for
-Flutter and Dart DevTools
+The following is an index of release notes for
+select DevTools releases.
+
+{{site.alert.note}}
+  DevTools is bundled as part of the Dart and Flutter SDKs.
+  To determine which version of DevTools you are using,
+  run `dart devtools --version`.
+{{site.alert.end}}
```

Comment:
**issue:** This reads awkwardly. Consider something that mirrors the [Golang release notes](https://go.dev/doc/devel/release).

```suggestion
This page summarizes the changes between official stable releases of the DevTools.
To view a complete list of changes, check out the
[DevTools git log](https://github.com/flutter/devtools/commits/master).

The Dart and Flutter SDKs include the DevTools.
To check your version of DevTools, run the following on your command line:

```terminal
dart devtools --version
```
```